nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java

```diff
@@ -123,12 +126,34 @@ public Object run(final NotificationServiceConfiguration notificationServiceConf
                         applyValues(c);
                         notificationServiceConfigurationProxy.getNotifierConfigurationList().add(c);
                         actionReport.setActionExitCode(ActionReport.ExitCode.SUCCESS);
+
+                        ParameterMap params = new ParameterMap();
+                        params.add("enabled", Boolean.TRUE.toString());
+                        params.add("dynamic", Boolean.TRUE.toString());
```

Comment:
Sorry, don't think I explained it properly.
I'd like these to be set to the parameters passed in from the command line, so something like:

```
params.add("enabled", enabled.toString());
if (dynamic != null) {
    params.add("dynamic", dynamic.toString());
}